app/src/main/java/com/example/keniphhomemaintenance/ui/cookbook/KeniphOutlinedTextField.kt

```diff
@@ -0,0 +1,41 @@
+package com.example.keniphhomemaintenance.ui.cookbook
+
+import androidx.compose.foundation.layout.fillMaxWidth
+import androidx.compose.foundation.text.KeyboardActions
+import androidx.compose.foundation.text.KeyboardOptions
+import androidx.compose.material.OutlinedTextField
+import androidx.compose.material.Text
+import androidx.compose.runtime.Composable
+import androidx.compose.runtime.MutableState
+import androidx.compose.ui.Modifier
+import androidx.compose.ui.focus.FocusDirection
+import androidx.compose.ui.focus.FocusManager
+import androidx.compose.ui.text.input.ImeAction
+import androidx.compose.ui.text.input.KeyboardType
+
+@Composable
+fun KeniphOutlinedTextField(
+    textMutableState: MutableState<String>,
+    textLabel: String,
+    focusManager: FocusManager,
```

Comment:
Can you get the instance of the FocusManager from within this composable instead of having to pass it in?

Reply:
ye